### PR TITLE
Wrap assembly comments consistently

### PR DESCRIPTION
**Pull request by mgeisler**
Some lines were much longer than others. They are now consistently
wrapped at 70 columns. Why so short, why not 80 or 90? Because we
sometimes need to browse these files during class, and there we don’t
have a lot of horizontal space when using large fonts.

### CODE REVIEW

File: src/bare-metal/aps/examples/exceptions.S [hunk 1]
```diff
@@ -64,7 +65,9 @@
 	ldr x18, [sp, #8 * 18]
 	ldp x29, x30, [sp, #8 * 20]
 
-	/* Restore registers elr_el1 & spsr_el1, using x0 & x1 as scratch. */
+	/*
+     * Restore registers elr_el1 & spsr_el1, using x0 & x1 as scratch.
```
Comment (michael-kerscher) [inline src/bare-metal/aps/examples/exceptions.S:69]: this comment seems to have an incorrect indentation

Comment (michael-kerscher) [reply src/bare-metal/aps/examples/exceptions.S:69]: it looks like tabs vs. spaces

File: src/bare-metal/aps/examples/exceptions.S [hunk 2]
```diff
@@ -64,7 +65,9 @@
...
+     */
```
Comment (mgeisler) [inline src/bare-metal/aps/examples/exceptions.S:70]:
```suggestion
	/*
	 * Restore registers elr_el1 & spsr_el1, using x0 & x1 as scratch.
	 */
```

File: src/bare-metal/aps/examples/image.ld [hunk 3]
```diff
@@ -67,7 +67,10 @@ SECTIONS
 		data_end = .;
 	} >image
 
-	/* Everything beyond this point will not be included in the binary. */
+	/*
+     * Everything beyond this point will not be included in the
+     * binary.
+     */
```
Comment (mgeisler) [inline src/bare-metal/aps/examples/image.ld:73]:
```suggestion
	/*
	 * Everything beyond this point will not be included in the
	 * binary.
	 */
```

File: src/bare-metal/aps/examples/exceptions.S (hunk 1 above)
Comment (mgeisler) [reply src/bare-metal/aps/examples/exceptions.S]: Oh my, thanks for spotting this. I don't know why the comment was indented using a tab originally — I've now indented the other comments with tabs too.

File: src/bare-metal/aps/examples/entry.S [hunk 4]
```diff
@@ -72,18 +78,20 @@
...
+ * these may contain boot parameters.
  */
 // ANCHOR: entry
 .section .init.entry, "ax"
 .global entry
 entry:
     /*
-     * Load and apply the memory management configuration, ready to enable MMU and
-     * caches.
+     * Load and apply the memory management configuration, ready to
```
Comment (michael-kerscher) [inline src/bare-metal/aps/examples/entry.S:93]: these are spaces to indent, files below have tabs. Should we try to have one style over all files?
This could also be a separate commit

**Review by michael-kerscher — APPROVED** [review]
The commit looks good to me. There are differences in the indentation style (tabs vs spaces) though.

File: src/bare-metal/aps/examples/entry.S (hunk 4 above)
Comment (mgeisler) [reply src/bare-metal/aps/examples/entry.S:93]: Yeah, having a consistent style would be good! Perhaps we could teach `dprint` to do this, but it might not be worth is since we touch these files very very rarely.
